docs/api-reference/create-next-app.md

```diff
@@ -32,7 +32,7 @@ pnpm create next-app -- --ts
 - **-e, --example [name]|[github-url]** - An example to bootstrap the app with. You can use an example name from the [Next.js repo](https://github.com/vercel/next.js/tree/canary/examples) or a GitHub URL. The URL can use any branch and/or subdirectory.
 - **--example-path [path-to-example]** - In a rare case, your GitHub URL might contain a branch name with a slash (e.g. bug/fix-1) and the path to the example (e.g. foo/bar). In this case, you must specify the path to the example separately: `--example-path foo/bar`
 - **--use-npm** - Explicitly tell the CLI to bootstrap the app using npm. To bootstrap using yarn we recommend running `yarn create next-app`
-- **--use-pnpm** - Explicitly tell the CLI to bootstrap the app using pnpm. To bootstrap using yarn we recommend running `yarn create next-app`
+- **--use-pnpm** - Explicitly tell the CLI to bootstrap the app using pnpm. To bootstrap using pnpm we recommend running `pnpm create next-app`
```

Comment:
```suggestion
- **--use-pnpm** - Explicitly tell the CLI to bootstrap the app using pnpm

Note: To bootstrap using `yarn` we recommend running `yarn create next-app`
```
This was meant to explain how `yarn` can be used with bootstrapping since there isn't a `--use-yarn` flag and is why it is also listed for the `--use-npm` note above. 

This is a bit repetitive so might be better to remove this note from both of those lines and just have a note underneath